meta-lmp-bsp/conf/machine/uz3eg-iocc.conf

```diff
@@ -3,6 +3,7 @@
 #@DESCRIPTION: Machine support for UltraZed-EG with IOCC customized for LmP
 
 DEFAULTTUNE = "cortexa53"
+SOC_VARIANT = "eg"
 
 require conf/machine/include/ultrazed-lmp.inc
```

Comment:
ok, found reply for my question from above.

Maybe we can replace it with `require conf/machine/include/zynqmp-lmp.inc` instead?

Reply:
Yup, planning on following that up on another pr.